### ISSUE TITLE
Feature request: export replicaSet for mongoDB

### ISSUE THREAD

**Issue by chanind** [terraform-providers/terraform-provider-alicloud#1928]
The mongo DB created using `alicloud_mongodb_instance` currently exports only the `id` of the resource, but no connection information. It seems like it's possible to guess the hostnames of the primary/secondary mongo from the instance id, but there's no way to know the `replicaSet`. The `replicaSet` is shown in the UI, so clearly it's possible to know this info, but without exporting the `replicaSet` isn't not possible to connect to the created DB using a lot of mongo clients.

**Comment by rsuhendro** [comment]
Pls bear with me but how do you get the connection string and port from the replicaSet? For the mongoDB Shard it is clearly stated in the doc, they are in the mongo_list. But the doc only states replicaSet but no reference to connection string and port. I appreciate any pointer, thanks..

**Comment by chanind** [comment]
It's super hacky, and I wish the mongo provider would just return the connection string directly. Basically it seems like the pattern is:
Port: 3717
and hosts are: "${alicloud_mongodb_instance.db.id}${count.index + 1}.mongodb.rds.aliyuncs.com"
so, take the ID of the DB, add `1` or `2` after it, and then `.mongodb.rds.aliyuncs.com`.
I don't know if this pattern is guaranteed to always work, or maybe I just got lucky for the DBs I created.
